cli/cli/kurtosis_config/overrides_objects/v5/logs_aggregator_config_v5.go

```diff
@@ -0,0 +1,5 @@
+package v5
+
+type LogsAggregatorConfigV5 struct {
```

Comment:
Thought while reading: you'll very likely want to update user-facing documentation with the latest info on how to configure the various new things added in this PR

Reply:
nice will add some docs